### PR TITLE
Remove RSS icon from the header.

### PR DESCRIPTION
**Pull request by KrinsKumar**
This PR address issue - #723.

I have removed the icon and its logic from the header component for now.

I have not yet removed `functions/api/share/[[user_id]].ts` logic which is used to generate the RSS feed. Please let me know if you want this to be removed. 

image

Thank you.

### CODE REVIEW

**Review by humphd — CHANGES_REQUESTED** [review]
You need to rip out the backend code in functions/ too.  Go look at the PRs that added all this.

**Review by humphd — CHANGES_REQUESTED** [review]
No, you've removed all the sharing backend logic.  Please only remove the RSS feed code.

**Comment by KrinsKumar** [comment]
Apologies, I have added the files back and removed the function used to generate the feed. Please let me know if this works. Thank you.

**Review by humphd — CHANGES_REQUESTED** [review]
Need to fix the lint issues https://github.com/tarasglek/chatcraft.org/actions/runs/11923809412/job/33296868143?pr=744

File: functions/api/share/[[user_id]].ts [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import { Feed } from "feed";
-import { DOMParser } from "linkedom";
```
Comment (humphd) [inline functions/api/share/[[user_id]].ts:2]: If you're removing these, can you confirm that we still need the dependencies, and if we don't `import` them anywhere else, please remove from `package.json` as well?

Comment (KrinsKumar) [reply functions/api/share/[[user_id]].ts:2]: I dont see us using DOMparsing from linkedom or feed anywhere else, I have removed them from `package.json`.

**Review by humphd — APPROVED** [review]

**Comment by humphd** [comment]
Thanks for all the work on this, @KrinsKumar!

**Pull request merged**